public/css/main.css

```diff
@@ -0,0 +1,760 @@
+@import url("https://fonts.googleapis.com/css?family=Dosis:400,600,700");
```

Comment:
You dont need to check in this file, since the corresponding scss files are already added

Reply:
Done.